### PR TITLE
Feature/marketplace redesign

### LINKED ISSUE #611: Enhancement: Redesign Marketplace
### Application

Marketplace

### Dependency

No

### Feature Description

Let's come up with a new design for this shit app.

### References

_No response_

### PR DESCRIPTION
**Pull request by antonstjernquist**
Redesigned marketplace. 

New features:
* Ability to edit listing
* Set price

**Pull Request Checklist**:
* [X] Have you followed the guidelines in our Contributing document and Code of Conduct?
* [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/project-error/npwd/pulls) for the same update/change?
* [X] Have you built and tested NPWD in-game after the relevant change?

### CODE REVIEW

**Comment by itschip** [comment]
Scope creeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeep

**Comment by antonstjernquist** [comment]
> Scope creeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeep

Removed some changes regarding useConfig, config.json and similar that wasn't really needed in this PR!

**Comment by itschip** [comment]
@antonstjernquist CI test failed. If you resolve this, we can moorge.

**Comment by itschip** [comment]
Will review new changes tonight.

File: phone/src/apps/marketplace/components/Home/MarketplaceSort.tsx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+import SelectUnstyled, {
+  SelectUnstyledProps,
+  selectUnstyledClasses,
+} from '@mui/base/SelectUnstyled';
+import OptionUnstyled, { optionUnstyledClasses } from '@mui/base/OptionUnstyled';
+import PopperUnstyled from '@mui/base/PopperUnstyled';
+import { styled } from '@mui/system';
+import { red, grey } from '@mui/material/colors';
+
+const StyledButton = styled('button')(
```
Comment (itschip) [inline phone/src/apps/marketplace/components/Home/MarketplaceSort.tsx:16]: Maybe moves these to a `*.styles` file?

**Comment by itschip** [comment]
@antonstjernquist It seems like you've added new column `price`. Make sure to update the `import.sql` as well.

**Comment by itschip** [comment]
REBASE PLZZZZ

**Pull request closed without merging**